main.go

```diff
@@ -833,6 +833,13 @@ func initialiseSystem() error {
 		if globalConf.PIDFileLocation == "" {
 			globalConf.PIDFileLocation = "/var/run/tyk/tyk-gateway.pid"
 		}
+		if globalConf.UseDBAppConfigs && globalConf.Policies.PolicySource != "service" {
+			globalConf.Policies.PolicySource = "service"
```

Comment:
might worth to introduce constants for for string values like `"service"` and `"tyk_policies"`, maybe in config package (not sure about that)